packages/widgets/docs-browser/src/components/DocsView/DocsView.js

```diff
@@ -108,7 +110,6 @@ const DocsView = props => {
 
   useEffect(() => {
     changeListParent(parent)
-    setSelection([])
```

Comment:
question(blocking): removing this line has no negative impact on the normal dms?

Reply:
Hmm I did not saw any issues.. I re-tested it an realized that we have a different behaviour now